src/model_repository_manager/model_lifecycle.cc

```diff
@@ -847,4 +860,19 @@ ModelLifeCycle::OnLoadFinal(
   }
 }
 
+void
+ModelLifeCycle::ReportModelLoadTime(
+    std::shared_ptr<MetricModelReporter> reporter,
+    const std::chrono::duration<double>& time_to_load)
+{
+#ifdef TRITON_ENABLE_METRICS
+  // Pending request count should always be 0 or 1 per-request. A request should
+  // not decrement the count unless it has already been incremented.
```

Comment:
Is this comment related or copied?

Reply:
Copied the usage of metrics reporter. Comment came along by mistake. Fixed.